frontend/src/tabs/Account.js

```diff
@@ -1,8 +1,11 @@
 import React, { useState, useEffect } from "react";
 import "./Account.css";
 import BasicTab from "./BasicTab.js";
+import backendConfig from "../configs/backend.config.json";
 
 const Account = (props) => {
+  const backendUrl = "http://" + backendConfig.host + ":" + backendConfig.port;
+  const getUsernameUrl = backendUrl + "/getUsername";
```

Comment:
get username needs to take the address as a query parameter, like `/getUsername?address={address}`, you can keep address as a 0 constant valid for now